GetLatestMigrationsStatus/index.ts

```diff
@@ -2,27 +2,18 @@ import express = require("express");
 import { setAppContext } from "@pagopa/io-functions-commons/dist/src/utils/middlewares/context_middleware";
 import createAzureFunctionHandler from "@pagopa/express-azure-functions/dist/src/createAzureFunctionsHandler";
 import { Context } from "@azure/functions";
-import { getApiClient } from "../utils/apim";
 import { getConfigOrThrow } from "../utils/config";
+import getPool from "../utils/db";
 import { getLatestMigrationsHandler } from "./handler";
 
 const config = getConfigOrThrow();
-
-// Get APIM Client
-const apimClient = getApiClient(
-  {
-    clientId: config.APIM_CLIENT_ID,
-    secret: config.APIM_SECRET,
-    tenantId: config.APIM_TENANT_ID
-  },
-  config.APIM_SUBSCRIPTION_ID
-);
+const client = getPool(config);
 
 const setupExpress = (): express.Express => {
```

Comment:
Why we do not apply standard express security configuration via @pagopa/io-functions-commons/dist/src/utils/express => secureExpressApp ?

Reply:
Thanks, fix with 7da83df